courses/free-courses-en.md

```diff
@@ -1525,6 +1525,7 @@
 ### System Design
 
 * [System Design](https://www.youtube.com/playlist?list=PLMCXHnjXnTnvo6alSjVkgxV-VH6EPyvoX) - Gaurav Sen
+* [System Design Fundamentals](https://www.youtube.com/playlist?list=PL55RiY5tL51q4D-B63KBnygU6opNPFk_q) - ByteByteGo
```

Comment:
The link is redirecting to a different playlist, not the one you specified. Change it as needed